src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,119 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 20;
+    private static final int NOT_VALID_AGE = 15;
+    private static final int NEGATIVE_AGE = -1;
+    private static final int LARGE_AGE = 234534255;
+    private static final Integer NULL_AGE = null;
+    private static final String VALID_LOGIN = "Antony";
+    private static final String ANOTHER_VALID_LOGIN = "Vlad";
+    private static final String NOT_VALID_NAME = "";
+    private static final String VALID_PASSWORD = "654321";
+    private static final String NOT_VALID_PASSWORD = "111";
+
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user = new User();
```

Comment:
No need to create extra object here because u will create it before each test case
`Private User user;`